src/test/java/uk/gov/justice/digital/client/dynamodb/DynamoDBClientTest.java

```diff
@@ -0,0 +1,156 @@
+package uk.gov.justice.digital.client.dynamodb;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.model.*;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.justice.digital.domain.model.DomainDefinition;
+import java.io.IOException;
+import java.util.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+
+class DynamoDBClientTest {
+
+
+    private static final Logger logger = LoggerFactory.getLogger(DynamoDBClientTest.class);
+
+    private static final long serialVersionUID = 1;
+    private static DynamoDBClient dynamoDBClient;
+    private static AmazonDynamoDB dynamoDB;
+    private static ObjectMapper mapper;
+
+
+    @BeforeEach
+    void setUp() {
+        dynamoDB = mock(AmazonDynamoDB.class);
+        mapper = mock(ObjectMapper.class);
+        dynamoDBClient = new DynamoDBClient(dynamoDB, mapper);
+    }
+
+    @AfterEach
+    void tearDown() {
```

Comment:
No need to provide an empty teardown method. Just include the code that is needed.